### PR TITLE
Codacy fix part 2

### PR DESCRIPTION
**Pull request by pragamj**
# Problem Statement
- Codacy_fix_phase2

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)
[collections_log.txt](https://github.com/Seagate/cortx-test/files/8804337/collections_log.txt)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/prov/test_prov_single_node.py [hunk 1]
```diff
@@ -80,8 +81,8 @@ def teardown_method(self):
     @pytest.mark.tags("TEST-19439")
```
Comment (ketanarlulkar) [inline tests/prov/test_prov_single_node.py:81]: `teardown_method` could be function. Remove if nothing to teardown.

File: tests/prov/test_post_deploy_validation.py [hunk 2]
```diff
@@ -184,25 +182,24 @@ def test_ntpconfg_validate_single_node(self):
...
             node_obj=self.nd_obj,
             time_server=ntp_time_server_val)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 3: Validated time_server in /etc/chrony.conf response = {}".format(
-            resp[1]))
+        LOGGER.info("Step 3: Validated time_server in /etc/chrony.conf response = %s", resp[1])
 
         set_timezone = (random.choice(
             [ii for ii in timezone_data if ii != ntp_time_zone_val]))
         set_timesrv_ip = (random.choice(
```
Comment (ketanarlulkar) [inline tests/prov/test_post_deploy_validation.py:199]: use of Standard pseudo-random generators on line 197 and 199 (random)

File: tests/prov/test_post_deploy_validation.py [hunk 3]
```diff
@@ -430,26 +420,23 @@ def test_ntpconfg_validate_multi_node(self):
                 exp_t_srv=self.ntp_data["srvnode-1"][self.ntp_keys[0]],
```
Comment (ketanarlulkar) [inline tests/prov/test_post_deploy_validation.py:420]: Line 407 CSM_USER prefer snake_case

**Review by ketanarlulkar — CHANGES_REQUESTED** [review]
Check comments

**Review by sonalk0209 — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**